commitizen/commands/init.py

```diff
@@ -260,7 +264,7 @@ def _ask_version_provider(self) -> str:
     def _ask_version_scheme(self) -> str:
         """Ask for setting: version_scheme"""
         default = "semver"
-        if self.project_info.is_python:
+        if self.project_info.is_python or self.project_info.has_uv_lock:
```

Comment:
Is there any case supported by `uv` where a `uv.lock` may exists without a `pyproject.toml`?
It seems that is currently unsupported: https://github.com/astral-sh/uv/issues/6722#issuecomment-2315254679
But I might be missing something

Reply:
AFAIK, no. 🤔 yep we probably could just remove this check. Let me update it now